README.md

```diff
@@ -95,56 +87,21 @@ let device: Device = instance
-
-```rust
-let queue_info = vk::DeviceQueueCreateInfo::builder()
-    .queue_family_index(queue_family_index)
-    .queue_priorities(&priorities);
+### Pointer chains
 
-let device_create_info = vk::DeviceCreateInfo::builder()
-    .queue_create_infos(std::slice::from_ref(&queue_info))
-    ...;
-```
+Ash provides helpers for assembling pointer chains. Use `.push_next()` on the struct which insert the argument as first member of the chain.
 
-Builders have an explicit lifetime, and are marked as `#[repr(transparent)]`.
 ```rust
-#[repr(transparent)]
-pub struct DeviceCreateInfoBuilder<'a> {
-    inner: DeviceCreateInfo,
-    marker: ::std::marker::PhantomData<&'a ()>,
-}
-impl<'a> DeviceCreateInfoBuilder<'a> {
-    //...
-    pub fn queue_create_infos(
-        mut self,
-        queue_create_infos: &'a [DeviceQueueCreateInfo],
-    ) -> DeviceCreateInfoBuilder<'a> {...}
-    //...
-```
-
-Every reference has to live as long as the builder itself. Builders implement `Deref` targeting their corresponding Vulkan struct, so references to builders can be passed directly
-to Vulkan functions.
-
-Calling `.build()` will **discard** that lifetime because Vulkan structs use raw pointers internally. This should be avoided as much as possible because this can easily lead to dangling pointers. If `.build()` has to be called, it should be called as late as possible. [Lifetimes of temporaries](https://doc.rust-lang.org/reference/expressions.html#temporary-lifetimes) are extended to the enclosing statement, ensuring they are valid for the duration of a Vulkan call occurring in the same statement.
+let mut variable_pointers = vk::PhysicalDeviceVariablePointerFeatures::default();
+let mut corner = vk::PhysicalDeviceCornerSampledImageFeaturesNV::default();
 
-
-### Pointer chains
-
-```rust
-let mut variable_pointers = vk::PhysicalDeviceVariablePointerFeatures::builder();
-let mut corner =
-    vk::PhysicalDeviceCornerSampledImageFeaturesNV::builder();
-;
-let mut device_create_info = vk::DeviceCreateInfo::builder()
+let mut device_create_info = vk::DeviceCreateInfo::default()
     .push_next(&mut corner)
     .push_next(&mut variable_pointers);
 ```
 
-Pointer chains in builders differ from raw Vulkan. Instead of chaining every struct manually, you instead use `.push_next` on the struct that you are going to pass into the function. Those structs then get *prepended* into the chain.
-
-`push_next` is also type checked, you can only add valid structs to the chain. Both the structs and the builders can be passed into `push_next`. Only builders for structs that can be passed into functions will implement a `push_next`.
-
+The templated argument of `.push_next()` only allows valid structs to extend a given struct (known as [`structextends` in the Vulkan registry](https://registry.khronos.org/vulkan/specs/1.3/styleguide.html#extensions-interactions), mapped to `Extends*` traits).
```

Comment:
```suggestion
The generic argument of `.push_next()` only allows valid structs to extend a given struct (known as [`structextends` in the Vulkan registry](https://registry.khronos.org/vulkan/specs/1.3/styleguide.html#extensions-interactions), mapped to `Extends*` traits).
```

Reply:
@Ralith sure thing, I've been weighing this exact wording and indeed opted for the wrong one, thanks!